build.py

```diff
@@ -1768,6 +1785,29 @@ def backend_build(
     cmake_script.blankln()
 
 
+def backend_clone(
+    be,
+    clone_script,
+    tag,
+    install_dir,
+    github_organization,
+):
+    repo_clone_dir = os.path.join(install_dir, "backends")
+
+    clone_script.commentln(8)
+    clone_script.comment(f"'{be}' backend")
+    clone_script.comment("Delete this section to remove backend from build")
+    clone_script.comment()
+    clone_script.mkdir(repo_clone_dir)
+    clone_script.cwd(repo_clone_dir)
+    clone_script.gitclone(backend_repo(be), tag, be, github_organization)
```

Comment:
Need to address this: 
Olga:
Regarding build.py, current (locally working approach) is these lines: https://github.com/triton-inference-server/server/blob/oandreeva_vllm_ci/build.py#L1283-L1295
+ build.py currently clones the whole github repo to `install/backends`, this would need to be adjusted slightly to not copy `ci` and `tools` folders

Reply:
Good find. Added. Looking online, there is no way to exclude subdirectories from a clone (or, inversely, only include some subdirectories), so I will simply force remove these extra directories for now. Long-term, we could look at whether we're copying these directories somewhere and could make sure we're only copying the directory with the Python-based backend itself.

Reply:
Why not clone into tmp_dir(repo-dir) as all other backends and only copy the model.py in the backend directory(install dir)?

Reply:
Good idea, thanks for talking about this. Done!